docs/cellprofiler.rst

```diff
@@ -2,49 +2,52 @@ Analyze OMERO data using a Jupyter Notebook
 ===========================================
 
 
-**Description**
----------------
+Description
+-----------
 
 We will demonstrate how to integrate CellProfiler and
 OMERO using the CellProfiler Python API and the OMERO Python API. We
 will use a Jupyter notebook to demonstrate the integration.
 
 We will show:
 
--  How to adjust an existing CellProfiler pipeline so that it can be used with OMERO.
+- How to adjust an existing CellProfiler pipeline so that it can be used with OMERO.
 
--  How load images from a Plate using the OMERO API.
+- How load images from a Plate using the OMERO API.
 
--  How to run CellProfiler using its Python API.
+- How to run CellProfiler using its Python API.
 
--  How to plot the results.
+- How to plot the results.
 
--  How to save the generated results back to OMERO as OMERO.table so they can be used later on by OMERO.parade.
+- How to save the generated results back to OMERO as OMERO.table so they can be used later on by OMERO.parade.
 
-**Resources**
--------------
+Resources
+---------
 
 We will use a CellProfiler example pipeline to analyse RNAi screening
 data from the Image Data Resource (IDR).
 
--  PercentPositive Pipeline \ https://cellprofiler.org/examples/
+- Example pipeline `PercentPositive Pipeline <https://cellprofiler.org/examples/>`_.
```

Comment:
The link leads to a page where there are all CP examples. None of them is called "PercentPositive..." - it is quite hard to find the right one. Maybe point to the direct download, or cite the Paragraph Specialized Pipelines, see screenshot (possibly I do not have the right one, but this is the point here)

![Screen Shot 2020-06-23 at 17 03 06](https://user-images.githubusercontent.com/2478303/85427362-8fffe480-b573-11ea-9b51-e0471e47715e.png)